account.go

```diff
@@ -61,37 +61,60 @@ const (
 
 // AccountParams are the parameters allowed during account creation/updates.
 type AccountParams struct {
-	Params
-	Country, Email, DefaultCurrency, Statement, BusinessName, BusinessUrl,
-	BusinessPrimaryColor, SupportPhone, SupportEmail, SupportUrl,
-	FromRecipient, PayoutStatement string
-	ExternalAccount                      *AccountExternalAccountParams
-	LegalEntity                          *LegalEntity
-	PayoutSchedule                       *PayoutScheduleParams
-	DebitNegativeBal, NoDebitNegativeBal bool
-	TOSAcceptance                        *TOSAcceptanceParams
-	Type                                 AccountType
+	Params               `form:"*"`
+	Country              string                        `form:"country"`
+	Email                string                        `form:"email"`
+	DefaultCurrency      string                        `form:"default_currency"`
+	Statement            string                        `form:"statement_descriptor"`
+	BusinessName         string                        `form:"business_name"`
+	BusinessUrl          string                        `form:"business_url"`
+	BusinessPrimaryColor string                        `form:"business_primary_color"`
+	SupportPhone         string                        `form:"support_phone"`
+	SupportEmail         string                        `form:"support_email"`
+	SupportUrl           string                        `form:"support_url"`
+	FromRecipient        string                        `form:"from_recipient"`
+	PayoutStatement      string                        `form:"payout_statement_descriptor"`
+	ExternalAccount      *AccountExternalAccountParams `form:"external_account"`
+	LegalEntity          *LegalEntity                  `form:"legal_entity"`
+	PayoutSchedule       *PayoutScheduleParams         `form:"payout_schedule"`
+	DebitNegativeBal     bool                          `form:"debit_negative_balances"`
+	NoDebitNegativeBal   bool                          `form:"debit_negative_balances,invert"`
+	TOSAcceptance        *TOSAcceptanceParams          `form:"tos_acceptance"`
+	Type                 AccountType                   `form:"type"`
 }
 
 // AccountListParams are the parameters allowed during account listing.
 type AccountListParams struct {
-	ListParams
+	ListParams `form:"*"`
 }
 
 // AccountExternalAccountParams are the parameters allowed to reference an
 // external account when creating an account. It should either have Token set
 // or everything else.
 type AccountExternalAccountParams struct {
-	Params
-	Account, Country, Currency, Routing, Token string
+	Params            `form:"*"`
+	Account           string `form:"account_number"`
+	AccountHolderName string `form:"account_holder_name"`
+	AccountHolderType string `form:"account_holder_type"`
+	Country           string `form:"country"`
+	Currency          string `form:"currency"`
+	Routing           string `form:"routing_number"`
```

Comment:
So I think I'd vastly prefer global explicitness over trying to guess property names because it's just so much more readily apparent as to what's going on. This is quite inline with how Go does things too, as you can see with `form`'s symmetry to the tags used by the built-in `json` package.